src/components/__tests__/action-button.spec.js

```diff
@@ -50,6 +50,57 @@ describe('action button', () => {
       await aButton.onclick();
       expect(button.removeAttribute).to.have.been.calledWith('disabled');
     });
+    it('should call scale and resetScale on mousedown/up', async () => {
+      const aButton = actionButton(defaults);
+      aButton.offsetHeight = 100;
+      aButton.offsetWidth = 200;
+      aButton.style = { transform: '' };
```

Comment:
Is this needed?

Reply:
So the style object is not added unless it is not rendered for real. Wanted to mock it the way it would look in production. In this certain test it would work with only an empty style object.